app/controllers/application_controller.rb

```diff
@@ -52,7 +53,7 @@ def load_current_user(payload)
   end
 
   def serialize_resource(resources, serializer, root = nil, extra = {} )
-    opts = { each_serializer: serializer, root: root }.merge(extra)
+    opts = { each_serializer: serializer, root: root, scope: current_user }.merge(extra)
```

Comment:
We are using `serialize_resource` method at many places, what is the impact of adding a scope here? Please add a justification here.

Reply:
By adding scope here we can access it throgh serializer and i have checked the method  serialize_resource it is still functions correctly